### PR TITLE
Release v4.6.0

### PR DESCRIPTION
**Pull request by superbaud**
Closes zcash/zcash#5436.

### CODE REVIEW

**Review by steven-ecc — DISMISSED** [review]
utACK

File: doc/release-notes/release-notes-4.6.0.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+`libzcash_script`
+-----------------
+
+Two new APIs have been added to this library (`zcash_script_legacy_sigop_count`
+and `zcash_script_legacy_sigop_count_precomputed`), for counting the number of
+signature operations in the transparent inputs and outputs of a transaction.
+The presence of these APIs is indicated by a library API version of 2.
+
+Updated RPCs
+------------
```
Comment (str4d) [inline doc/release-notes/release-notes-4.6.0.md:22]:
```suggestion
Updated RPCs
------------

- The `getblocktemplate` RPC method output now includes a `defaultroots` field,
  which provides various tree roots and block commitments matching the contents
  of the block template. If any part of the block template marked as `mutable`
  in the RPC method output is mutated, these roots may need to be recomputed.
  For more information on the derivation process, see the block header changes
  in [ZIP 244](https://zips.z.cash/zip-0244#block-header-changes).
```

File: doc/release-notes/release-notes-4.6.0.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+- Fixed an issue where `ERROR: spent index not enabled` would be logged
+  unnecessarily on nodes that have either insightexplorer or lightwalletd
+  configuration options enabled.
+
+- The `getmininginfo` RPC now omits `currentblockize` and `currentblocktx`
```
Comment (str4d) [inline doc/release-notes/release-notes-4.6.0.md:28]:
```suggestion
- The `getmininginfo` RPC now omits `currentblocksize` and `currentblocktx`
```

File: doc/release-notes/release-notes-4.6.0.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+Two new APIs have been added to this library (`zcash_script_legacy_sigop_count`
+and `zcash_script_legacy_sigop_count_precomputed`), for counting the number of
+signature operations in the transparent inputs and outputs of a transaction.
+The presence of these APIs is indicated by a library API version of 2.
+
+Updated RPCs
+------------
+
+- Fixed an issue where `ERROR: spent index not enabled` would be logged
+  unnecessarily on nodes that have either insightexplorer or lightwalletd
```
Comment (str4d) [inline doc/release-notes/release-notes-4.6.0.md:25]:
```suggestion
  unnecessarily on nodes that have either `-insightexplorer` or `-lightwalletd`
```

**Review by str4d — APPROVED** [review]
utACK after changes to release notes.

**Review by str4d — APPROVED** [review]
utACK 25278c312a9fff58135c5470a1a180f51906c437

**Review by daira — APPROVED** [review]
utACK

**Review by steven-ecc — APPROVED** [review]
utACK [25278c3 ](https://github.com/zcash/zcash/commit/25278c312a9fff58135c5470a1a180f51906c437)

**Pull request merged**